suitcase/tiff/tests.py

```diff
@@ -22,11 +63,9 @@ def test_export(tmp_path, example_data, stack_images):
     collector = example_data()
     artifacts = export(collector, tmp_path, file_prefix='',
                        stack_images=stack_images)
+    expected = create_expected(collector, stack_images)
 
     for filename in artifacts.get('stream_data', []):
         actual = tifffile.imread(str(filename))
-        if len(actual.shape) == 3:
-            for img in actual:
-                assert_array_equal(img, expected)
-        else:
-            assert_array_equal(actual, expected)
+        streamname = str(filename).split('/')[-1].split('-')[0]
```

Comment:
This won't pass on Windows. The most direct fix is to replace `'/'` with `os.sep`. However, this might be made more readable with the use of higher-level `os` path-parsing functions. Example:

```py
In [2]: import os.path

In [3]: os.path.basename('a/b/c.txt')
Out[3]: 'c.txt'
```

Reply:
fixed